include/internal/libspdm_common_lib.h

```diff
@@ -630,6 +641,20 @@ bool libspdm_verify_certificate_chain_hash(libspdm_context_t *spdm_context,
                                            const void *certificate_chain_hash,
                                            size_t certificate_chain_hash_size);
 
+/**
+ * This function verifies the public key hash.
+ *
+ * @param  spdm_context            A pointer to the SPDM context.
+ * @param  public_key_hash         The public key hash data buffer.
+ * @param  public_key_hash_size    size in bytes of the public key hash data buffer.
+ *
+ * @retval true  hash verification pass.
+ * @retval false hash verification fail.
+ **/
+bool libspdm_verify_public_key_hash(libspdm_context_t *spdm_context,
+                                    const void *public_key_hash,
+                                    size_t public_key_hash_size);
```

Comment:
`public_key_hash_size` isn't needed since this info is in the `spdm_context`.

Reply:
There is a comparison between `public_key_hash_size` and the `hash_size` from `spdm_context` in the hash verification process (just as same as certificate chain has verification process does), so the `public_key_hash_size` is necessary.